### PR TITLE
adding content for supporting publications modeling 

### PR DESCRIPTION
**Pull request by mbrush**
- updated descriptions of Publication class and publications slot.
- added a description for 'article', and exact mappings
- added classes for journal article, patent, preprint, web page, drug label
- added PMCID to prefix (already referenced in many mappings)

### CODE REVIEW

**Review by sierra-moxon — APPROVED** [review]

**Comment by sierra-moxon** [comment]
converted to draft while waiting for review tomorrow.

**Review by mikebada — APPROVED** [review]
I still think it's kind of odd to characterize things like, e.g., patents and parts of informational entities like figures and figure legends as publications, but I'm willing to defer to the consensus.

**Pull request merged**
